moped-database/migrations/1727471090930_update_generic_proj_ext/down.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE moped_components SET is_deleted = true WHERE component_name = 'Project Extent - Generic' AND line_representation = false;
```

Comment:
in the spirit of having the down migrations restore the DB to the previous state, i think the right move here would be to actually hard delete the component that the up migration created. obviously, that's going to fail if there are any `moped_proj_components` that reference the component, so here are a few ideas:
- just acknowledge that the down migration will not work without manual intervention by the dev to deal with the extant `moped_proj_components` that reference it
- write the down migration to also migrate the `moped_proj_components` to some other component type (like intersection improvement generic)
- write the down migration to hard delete `moped_proj_components` which use this component type, yolo style

in any case, i don't think this is a big deal so i'm going to approve your PR! 🤷